Assets/Mirror/Examples/AdditiveScenes/Scripts/PlayerController.cs

```diff
@@ -5,28 +5,21 @@ namespace Mirror.Examples.Additive
     [RequireComponent(typeof(CharacterController))]
     public class PlayerController : NetworkBehaviour
     {
-        public override void OnStartServer()
-        {
-            base.OnStartServer();
-            playerColor = Random.ColorHSV(0f, 1f, 1f, 1f, 0.5f, 1f);
-        }
-
-        [SyncVar(hook = nameof(SetColor))]
-        public Color32 playerColor = Color.black;
+        public CharacterController characterController;
```

Comment:
this has nothing to do with making RandomColor separate,  should be it's own PR

Reply:
reverted